patches/tModLoader/Terraria/GameContent/Biomes/Desert/DesertDescription.cs.patch

```diff
@@ -0,0 +1,13 @@
+--- src/Terraria/Terraria/GameContent/Biomes/Desert/DesertDescription.cs
++++ src/tModLoader/Terraria/GameContent/Biomes/Desert/DesertDescription.cs
+@@ -60,8 +_,9 @@
+ 		public static DesertDescription CreateFromPlacement(Point origin) {
+ 			Vector2 defaultBlockScale = DefaultBlockScale;
+ 			float num = (float)Main.maxTilesX / 4200f;
++			float numy = (float)Main.maxTilesY / 1200f; //Needed for esoteric world size support. Shouldn't affect vanilla gen at all.
```

Comment:
Have we verified this actually has no affect? Do seeds and stuff still generate the same?

Reply:
Just making sure.